pkg/api/pod/util_test.go

```diff
@@ -3643,6 +3643,7 @@ func TestDropPodLifecycleSleepAction(t *testing.T) {
 
 	for i, tc := range testCases {
 		t.Run(fmt.Sprintf("test_%d", i), func(t *testing.T) {
+			featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.33"))
```

Comment:
Is there a reason why this is not 1.34?

Reply:
The feature is locked at 1.34, so we can't set it disabled if it's 1.34 here.

Reply:
explained in more detail in https://github.com/kubernetes/kubernetes/pull/132595#discussion_r2184869364